### PR TITLE
cache-manager - adding disconnect to cache

### PR DESCRIPTION
**Pull request by TimothyIp**
**Please check if the PR fulfills these requirements**
- [x] Followed the [Contributing](https://github.com/jaredwray/cacheable/blob/main/CONTRIBUTING.md) guidelines.
- [x] Tests for the changes have been added (for bug fixes/features) with 100% code coverage.
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Adds the ability to disconnect from the relevant stores inside the cache

### CODE REVIEW

File: packages/cache-manager/README.md [hunk 1]
```diff
@@ -332,6 +332,18 @@ await cache.wrap('error', () => {
 
 See unit tests in [`test/wrap.test.ts`](./test/wrap.test.ts) for more information.
 
+## disconnect
+
+`disconnect(key): Promise<void>`
+
+Will disconnect from the relevant store(s).
```
Comment (TimothyIp) [inline packages/cache-manager/README.md:339]: Let me know if we should add a small disclaimer about using disconnect only when needed. Depending on the store, for example, redis. It is best practice to re-use connections if possible

Comment (jaredwray) [reply packages/cache-manager/README.md:339]: That would be good to add in launguage about understanding the Keyv driver you are using and when you should use disconnect.

Comment (jaredwray) [reply packages/cache-manager/README.md]: I added it.

**Comment by jaredwray** [comment]
@TimothyIp - thanks so much and will get this rolled out in the next couple weeks.

**Comment by TimothyIp** [comment]
This is awesome! Thanks!

**Pull request merged**